src/scheduled-tasks/VoteReminder.ts

```diff
@@ -11,12 +12,33 @@ interface VoteReminderTaskPayload {
 @ApplyOptions<ScheduledTask.Options>({ name: 'VoteReminderTask', customJobOptions: { removeOnComplete: true } })
 export class VoteReminderTask extends ScheduledTask {
 	public async run(payload: VoteReminderTaskPayload) {
-		const { memberId } = payload;
-		const user = await this.container.client.users.fetch(memberId).catch(() => null);
+		const user = await this.container.client.users.fetch(payload.memberId).catch(() => null);
 		if (!user) return;
-		await user.send({
-			content: `Hi, you can vote for Birthdayy again!`,
-			embeds: [generateDefaultEmbed(VoteEmbed)]
-		});
+
+		const t = getT('en-US');
```

Comment:
do we have a way to get the users language from their id and send the embed in this language?

Reply:
https://github.com/BirthdayyBot/BirthdayyBot/pull/568/commits/9f156de2494bdcd807dacbea99e918f4ead5a60f